src/__tests__/stubs.tsx

```diff
@@ -64,6 +86,59 @@ const Table = React.forwardRef<CollectionRef, TableProps>(
     React.useImperativeHandle(ref, () => ({
       scrollToTop,
     }));
+    const { getItemExpandable, expandedItems = [], getItemChildren, onExpandableItemToggle } = expandableRows ?? {};
+
+    function TableItem({ item, itemIndex, parentIndex }: { item: Item; itemIndex: number; parentIndex?: string }) {
+      const isExpandable = getItemExpandable?.(item) ?? false;
+      const isExpanded = expandedItems.some(it => getTrackableValue(trackBy, it) === getTrackableValue(trackBy, item));
+      const nestedItems = getItemChildren?.(item) ?? [];
+      const dataIndex = firstIndex ? (!parentIndex ? `${firstIndex + itemIndex}` : `${parentIndex}-${itemIndex}`) : '';
+      return (
+        <div
+          key={item.id}
```

Comment:
Minor: do we need the `key` attribute here? It is already defined in the respective instances of this component in lines 137 and 168.